docs/application/native/guides/multimedia/media-playback.md

```diff
@@ -71,6 +71,10 @@ The following figure illustrates what happens when the player gets interrupted b
 
 ![Player states when interrupted by system](./media/using_players_interrupted_state.png)
 
+Software and hardware video codec can be selected using the player API. You should check the default codec type by calling `player_get_video_codec_type()`.
+
+Before you select the software codec, you should consider the performance of the hardware core because decoding speed and quality can be influenced by the performance of the hardware.
+
```

Comment:
@eunhaechoi I think it would be more appropriate if these lines are placed in between Line 68 and Line 70.